### PR TITLE
[OSDEV-1004] Setup OpenSearch and Logstash for the local development

### PR DESCRIPTION
**Pull request by vladsha-dev**
[OSDEV-1004](https://opensupplyhub.atlassian.net/browse/OSDEV-1004)
- Prepared the local environment setup for the Logstash and OpenSearch services to enable local development.
- Created a script to start the project from scratch with a database populated with sample data.

### CODE REVIEW

File: scripts/start_local_dev [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        echo "Waiting 60 seconds for services to be fully up and running..."
+        sleep 60
+
+        # Launch deduplication process of seeded lists to create new facilities.
+        echo "Launching deduplication process of seeded lists..."
+        ./scripts/manage matchfixtures
+
+        # Wait for the matching to finish.
+        echo "Waiting 60 seconds for matching to finish..."
+        sleep 60
```
Comment (roman-stolar) [inline scripts/start_local_dev:41]: Minor question: Is it enough 60 sec for matching?

Comment (vladsha-dev) [reply scripts/start_local_dev:41]: I checked that for seeded data(927 facilities) it is enough.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
Tested locally, works as exptected. Good work, I approve this PR.

**Pull request merged**